pkg/ocm/ocm.go

```diff
@@ -432,3 +433,46 @@ func (c *SdkClient) listLimitedSupportReasons(internalClusterID string) ([]*v1.L
 	}
 	return reasons.Items().Slice(), nil
 }
+
+func (c *SdkClient) getOrgfromClusterID(cluster *v1.Cluster) (string, error) {
+	subID, ok := cluster.Subscription().GetID()
+	if !ok {
+		return "", fmt.Errorf("failed getting subcription id from cluster object")
+	}
+
+	resp, err := c.conn.AccountsMgmt().V1().Subscriptions().List().Search(fmt.Sprintf("id like '%s'", subID)).Size(1).Send()
```

Comment:
I was thinking it'd be simpler to use the following instead:
`resp, err := c.conn.AccountsMgmt().V1().Subscriptions().Subscription(subID).Get().Send()`
That way, we can remove the checks (slice, length etc...) below. WDYT?

Reply:
That indeed would have been easier, thanks for pointing that out. 

I refactored to use the different api endpoint and base the check on the cluster only (not org).